ui.html

```diff
@@ -131,20 +134,24 @@ <h1 class="title">
 
     // 複数の要素が選択された際の処理
     if (type === "selectMultiple") {
```

Comment:
@ics-ebie 
こちら、「テキストが選択されない場合の処理」というのがわかるようなtypeとコメントに変更することは可能でしょうか？

例
```suggestion
    // テキストが選択されていない際の処理
    if (type === "hasNotTextLayer") {
```

Reply:
@ics-furudate 
こちらアラートの表示条件が更新できておらず、「何も選択していない時」に表示するものでしたので以下のように修正しました
```
    // 何も選択していない場合
    if (type === "hasNothing") {
```

Reply:
@ics-ebie ありがとうございます！確認しました！